app/selfserve/module/Olcs/src/Controller/Lva/AbstractUploadEvidenceController.php

```diff
@@ -358,4 +358,28 @@ private function shouldShowSupportingEvidence()
         }
         return $this->application['status']['id'] === RefData::APPLICATION_STATUS_UNDER_CONSIDERATION;
     }
+
+    /**
+     * Has any evidence been uploaded
+     */
+    function hasEvidence($data): bool
+    {
+        // Check if "financialEvidence" exists and its "fileCount" array is not empty
+        if (!empty($data['financialEvidence']['files']['fileCount'])) {
+            return true;
+        }
+
+        // Check if "operatingCentres" exists and is not empty
+        if (!empty($data['operatingCentres'])) {
+            return true;
+        }
+
+        // Check if "supportingEvidence" exists and its list array is not empty
+        if (!empty($data['supportingEvidence']['files']['list'])) {
+            return true;
+        }
+
+        // If all conditions fail, return false
+        return false;
+    }
```

Comment:
early returns so only has financial evidence covered if that is added, assume not clear on ticket that is OK?  Have nothing to block this goign in as in .. but other stylistic approach would maybe to drop required into array and loop through all fields and yield true on !empty condition